lib/edge_detection.dart

```diff
@@ -5,12 +5,31 @@ import 'package:flutter/services.dart';
 class EdgeDetection {
   static const MethodChannel _channel = const MethodChannel('edge_detection');
 
-  static Future<String?> get detectEdge async {
-    final String? imagePath = await _channel.invokeMethod('edge_detect');
-    return imagePath;
+  static Future<bool> detectEdge(String saveTo,
+      {bool canUseGallery: true,
+      String androidScanTitle: "Scanning",
+      String androidCropTitle: "Crop",
+      String androidCropBlackWhiteTitle: "Black White",
+      String androidCropReset: "Reset"}) async {
+    return await _channel.invokeMethod('edge_detect', {
+      'save_to': saveTo,
+      'can_use_gallery': canUseGallery,
+      'scan_title': androidScanTitle,
+      'crop_title': androidCropTitle,
+      'crop_black_white_title': androidCropBlackWhiteTitle,
+      'crop_reset_title': androidCropReset,
+    });
   }
-  static Future<String?> get detectEdgeFromGallery async {
-    final String? imagePath = await _channel.invokeMethod('edge_detect_gallery');
-    return imagePath;
+
+  static Future<bool> detectEdgeFromGallery(String saveTo,
+      {String androidCropTitle: "Crop",
+      String androidCropBlackWhiteTitle: "Black White",
+      String androidCropReset: "Reset"}) async {
+    return await _channel.invokeMethod('edge_detect_gallery', {
+      'save_to': saveTo,
+      'crop_title': androidCropTitle,
+      'crop_black_white_title': androidCropBlackWhiteTitle,
+      'crop_reset_title': androidCropReset,
+    });
```

Comment:
@VoronovAlexander are we not supposed to pass 
'from_gallery' : true
 for detectEdgeFromGallery to work properly?